k8s/gogs/chart/gogs/templates/application.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    kubernetes-engine.cloud.google.com/icon: >-
+      data:image/png;base64,{{ .Files.Get "logo.png" | b64enc }}
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "click-to-deploy-containers", "product_id": "gogs", "partner_name": "Google Click to Deploy"}'
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name }}"
+spec:
+  descriptor:
+    type: Gogs
+    version: "{{ .Values.gogs.image.tag }}"
+    description: |-
+      Gogs is a lorem ipsum.
+      # Support
+      Google does not offer support for this solution. However, community support is available on
+      [Stack Overflow](https://stackoverflow.com/questions/tagged/conjur/).
+    maintainers:
+    - name: Google Click to Deploy
+      url: https://cloud.google.com/solutions/#click-to-deploy
+    links:
+    - description: 'User Guide: Google Click to Deploy Conjur'
+      url: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/conjur/README.md
+    notes: |-
+      # Access Conjur
+      {{ if .Values.enablePublicServiceAndIngress -}}
+      # Open your Conjur Webserver
+      To get the external IP of your Conjur Webserver, use the following command:
+
+      ```
+      SERVICE_IP=$(kubectl get ingress {{ .Release.Name }}-conjur-svc \
+        --namespace {{ .Release.Namespace }} \
+        --output jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+      echo "https://${SERVICE_IP}/"
+      ```
+      {{- else -}}
+      # Open your Conjur Webserver
```

Comment:
conjur

Reply:
?

Reply:
i'll fix it, file was copied from conjur solution